packages/select/src/vaadin-lit-select.js

```diff
@@ -37,8 +36,7 @@ class Select extends SelectBaseMixin(ElementMixin(ThemableMixin(PolylitMixin(Lit
 
   static get styles() {
     return [
-      fieldShared,
-      inputFieldContainer,
+      inputFieldShared,
```

Comment:
Select was previously not importing `clearButton` style module as it doesn't have a clear button.
Now it's changed to also covers toggle button, so I updated select to use the shared module.

Reply:
Thanks, I didn't realize and didn't check all the components, just assumed. I think it’s not a big deal that Select now also gets the clear button styles.